### ISSUE TITLE
When there is a blank before the phone number gTalkSMS says the SMS wasn't send

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#170]
```

1) my command
moi:  sms: 33699999999:As tu bien renvoyé le document que je viens de te 
renvoyer

2) gTalkSMS answer
reply:  SMS envoyé à Laurent YYYYYY: "As tu bien renvoyé  le documen..."
SMS "As tu bien renvoyé le documen..." pour Laurent YYYYYY non envoyé.
Le destinataire par défaut est Laurent YYYYYY (33699999999)

-> non envoyé

je sams command without the blank says "envoyé"

gTalkSMS should be able to trim the blanksin that case. 
```

Original issue reported on code.google.com by `lpenou.tstXX@mpo.li` on 17 May 2011 at 1:27

**Comment by GoogleCodeExporter** [comment]
```
It works for me with "sms: 06XXXXXXXX : hello" but if I do "sms : 06XXXXXXXX : 
hello" the command isn't reconized.

I'll do a trim on command names.
```

Original comment by `Florent....@gmail.com` on 10 Jun 2011 at 9:12

**Comment by GoogleCodeExporter** [comment]
```
Trim on command name done
```

Original comment by `Florent....@gmail.com` on 10 Jun 2011 at 9:17
- Changed state: **WaitingForRelease**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in v2.2.4
```

Original comment by `Florent....@gmail.com` on 13 Jun 2011 at 1:24
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 25 Aug 2011 at 7:50
- Changed state: **Deployed**
